packages/common/src/utils/is-update-group.ts

```diff
@@ -0,0 +1,12 @@
+/* Copyright (c) 2020 Environmental Systems Research Institute, Inc.
+ * Apache-2.0 */
+
+import { IGroup } from "@esri/arcgis-rest-types";
+
+/**
+ * Determines if a given IGroup is an update group
+ * @param {IGroup} group The group to evaluate
+ */
+export const isUpdateGroup = (
+  group: IGroup
+): boolean => group.capabilities.indexOf("updateitemcontrol") > -1;
```

Comment:
Why not `.includes`?

Reply:
> Why not `.includes`?

Typically I do, I _thought_ the TS compiler was complaining about `includes`, but I just built without issue. Will change.